examples/step-1/step-1.cc

```diff
@@ -196,7 +194,7 @@ void second_grid ()
       // <a href="http://en.cppreference.com/w/cpp/language/range-for">range-
       // based for loops</a>, which wrap up all of the syntax shown above into a
       // much shorter form:
-      for (auto cell: triangulation.active_cell_iterators())
+      for(auto cell : triangulation.active_cell_iterators())
```

Comment:
Hm, I think I liked the previous non-attached opening parenthesis better.

(But the space before the colon is definitely worth it.)

Reply:
Well, we have an option to control spaces before control statements (separately from function calls and declarations/definitions). I always use the non-space version in my own code, i.e. `if(...)`. Guess we have to fight a long and bitter fight about what we want :stuck_out_tongue_closed_eyes:
(Discussing the correct options for `.clang-format` has definitely the potential for a gigantic bikeshedding.)

Reply:
That is easy to change for control statements. All the other cases (function calls, function declarations) are treated uniformly.

Reply:
> I think I liked the previous non-attached opening parenthesis better.

same here.

Reply:
Compare masterleinad#12